examples/guided.py

```diff
@@ -410,8 +408,6 @@ def perform_installation(mountpoint):
 	# Temporarily disabling keep_partitions if config file is loaded
 	archinstall.arguments['harddrive'].keep_partitions = False
 	# Temporary workaround to make Desktop Environments work
-	archinstall.storage['_desktop_profile'] = archinstall.arguments.get('desktop', None)
-	if archinstall.arguments.get('profile', None):
-		archinstall.arguments['profile'] = Profile(installer=None, path=archinstall.arguments['profile']['path'])
```

Comment:
Okay, so the reason to remove this logic was that the user would have to provide the actual path to the profile in the config file, which is not very user friendly, the new logic simply picks up a key `desktop` from the config file and uses it to install whatever profile is specified, but yeah this is very confusing, I'll update the PR with your suggested changes, I missed that the path can actually be just the profile name, my bad.

Reply:
Actually, just renaming "desktop" to "profile" will also work, AFAIU

Reply:
Just for the sake of keeping the UX simple, i'll just rename the key to "profile" for now, because asking users to put in
`profile: {
 path: ""
}
`
would be a lot messier

we could keep it as simple as
`profile: <name>`

Reply:
The user doesn't have to provide an actual path, we have path-lookup mechanisms, so any short version of profiles would work. We support the following from the user currently:
 * Partial
 * Absolute
 * Remote
 I'm guessing even when loading from a configuration, this should be the case as well :) But first implementation can be limited, that's fine :)

Reply:
tested with all three